### ISSUE TITLE
Dopamine - Not Showing in Settings

### ISSUE THREAD

**Issue by jobsgonnawork** [Paisseon/Erika#19]
So I have recently installed Erika, everything ok, but when I open settings and scroll down to the tweaks, no sign of Erika whatsoever. I can’t enable it.

**Comment by Paisseon** [comment]
Doesn't have prefs. Just tap the tweak icon in Sileo and it will popup

**Comment by Mabdelwanis** [comment]
are it compatible with dopamine?

**Comment by Paisseon** [comment]
Yes

**Comment by jebediah2** [comment]
> Doesn't have prefs. Just tap the tweak icon in Sileo and it will popup

I'm not seeing any Erika tweak icon in Dopamine's Sileo either. Is it supposed to be somewhere I'm not looking?

**Comment by Paisseon** [comment]
> > Doesn't have prefs. Just tap the tweak icon in Sileo and it will popup
> 
> I'm not seeing any Erika tweak icon in Dopamine's Sileo either. Is it supposed to be somewhere I'm not looking?

Icon of the tweak you want to download, not Erika's icon

**Comment by jebediah2** [comment]
Thanks a lot. I think it would be useful to explain how Erika works for those who never used it before. Else there's no way to know except by watching somebody else use it on youtube or something.

**Comment by Mabdelwanis** [comment]
> Thanks a lot. I think it would be useful to explain how Erika works for those who never used it before. Else there's no way to know except by watching somebody else use it on youtube or something.

Are it worked for you ? Still can’t download any free deb file

**Comment by Paisseon** [comment]
It's in the depiction. And Erika mostly doesn't work on free tweaks, but if you want to check if it works, try on Crane.

**Comment by Mabdelwanis** [comment]
Tested on crane worked but any other paid tweak on havoc not work 
Tested on genesis 3
![482A9DAA-0D5B-447A-B760-8630011F6E48](https://github.com/Paisseon/Erika/assets/3472774/e88c8ca7-002f-4431-a2dd-4f509cef8f16)

**Comment by jebediah2** [comment]
> It's in the depiction. And Erika mostly doesn't work on free tweaks, but if you want to check if it works, try on Crane.

That's correct. I guess I overlooked it and/or got confused by that instruction. Also, the correct word to use would be 'description'. But thanks, I figured ut now.

> Tested on crane worked but any other paid tweak on havoc not work Tested on genesis 3

If you read Erika's 'Known Issues' section in the description, it says 'not found on the server' means that tweak is not featured yet by Erika. So your Erika installation is working, just the tweak you wanted isn't yet provided.
